activerecord/test/cases/finder_test.rb

```diff
@@ -572,6 +572,20 @@ def test_take_and_first_and_last_with_integer_should_use_sql_limit
     assert_sql(/LIMIT|ROWNUM <=/) { Topic.last(5).entries }
   end
 
+  def test_last_with_offset
+    topic = Topic.order(:id)
+    assert_equal topic.to_a.last, topic.last
+    assert_equal topic.offset(1).to_a.last, topic.offset(1).last
+    assert_equal topic.last, topic.offset(4).last
+  end
```

Comment:
Shouldn't this one be `assert_equal topic.offset(4).to_a.last, topic.offset(4).last`?